.github/workflows/build_test.yaml

```diff
@@ -47,3 +47,9 @@ jobs:
           cd substrate-node/tests
           robot -d _output_tests/ .
 
+      - uses: actions/upload-artifact@v3
+        if: failure()
```

Comment:
Maybe we can set this to always()? In cases where we want to check specific logs from even successful jobs?